felix/config/config_params.go

```diff
@@ -331,6 +331,14 @@ type Config struct {
 	EndpointReportingEnabled   bool          `config:"bool;false"`
 	EndpointReportingDelaySecs time.Duration `config:"seconds;1"`
 
+	// EndpointStatusPathPrefix is the path to the directory
+	// where endpoint status will be written. Endpoint status
+	// file reporting is disabled if field is left empty.
+	//
+	// Chosen directory should match the directory used by the CNI for PodStartupDelay.
+	// [Default: empty]
+	EndpointStatusPathPrefix *string `config:"file;;"`
```

Comment:
```suggestion
	EndpointStatusPathPrefix string `config:"file;;"`
```
Normally just use string instead of `*string`. Only need a `*string` if nil and "" have distinct meanings, which we should avoid unless its forced.

Reply:
Done